packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -302,16 +243,15 @@ const DMT5TradeModal = ({
                         <Icon icon='IcInstallationHuawei' width={135} height={40} />
                     </a>
                 </div>
-                {!isMobile() && (
-                    <div className='cfd-trade-modal__download-center-options--qrcode'>
-                        <img src={getUrlBase('/public/images/common/mt5_download.png')} width={80} height={80} />
-                        <Text align='center' size='xxs'>
-                            {localize('Scan the QR code to download {{ platform }}.', {
-                                platform: getPlatformSettings('mt5').name,
-                            })}
-                        </Text>
-                    </div>
-                )}
+
+                <div className='cfd-trade-modal__download-center-options--qrcode'>
```

Comment:
```suggestion
                <div className='cfd-trade-modal__download-center-options--qrcode-hide'>
```

Reply:
It is already being handled by @mobile mixin